crates/tinymist-query/src/analysis/linked_def.rs

```diff
@@ -100,19 +100,34 @@ pub fn find_definition(
                 .or_else(|| {
                     let sel = Selector::Label(label);
                     let elem = introspector.query_first(&sel)?;
-                    let span = elem.span();
-                    let fid = span.id()?;
 
-                    let source = ctx.source_by_id(fid).ok()?;
-
-                    let rng = source.range(span)?;
+                    // if it is a label, we put the selection range to itself
+                    let (fid, rng, name_range) = if is_label {
```

Comment:
nit:
i personally prefer `range`, `file_id` as variable name.

rng usuallly refers to random number generator

update: it looks like fid is actually a span id🤔

Reply:
`rng` is `range`, and `fid` is `file_id`.

Reply:
but there are 44 `fid` and 81 `rng` in the repository, if we are going to use longer name, we can fix them all once.

Reply:
> rng usuallly refers to random number generator

In the domain of program analysis, we are rarely use random number generator so there is no much misleading by using `rng` as `range`.

Reply:
> but there are 44 `fid` and 81 `rng` in the repository, if we are going to use longer name, we can fix them all once.

it's okay to ignore this comment if it is already widely used